.travis.yml

```diff
@@ -7,13 +7,9 @@ matrix:
   - rvm: jruby-9.1.13.0
     env: LOGSTASH_BRANCH=master
   - rvm: jruby-9.1.13.0
-    env: LOGSTASH_BRANCH=7.0
+    env: LOGSTASH_BRANCH=7.5
   - rvm: jruby-9.1.13.0
-    env: LOGSTASH_BRANCH=6.7
-  - rvm: jruby-9.1.13.0
-    env: LOGSTASH_BRANCH=6.6
-  - rvm: jruby-1.7.27
-    env: LOGSTASH_BRANCH=5.6
```

Comment:
do we maybe want to specify `required_ruby_version >= 2.0` in the gemspec (or even >= 2.3) ?
or alternatively logstash-core >= 6.8 ?